packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -28,18 +28,24 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
-      ),
+      // This is the theme of your application.
+      //
+      // Try running your application with "flutter run". You'll see the
+      // application has a purple toolbar. Then, without quitting the app,
+      // try changing the primary color of the theme to green by modifying
+      // the line below to:
+      //
+      // theme: ThemeData.from(colorScheme: ColorScheme.light(primary: Colors.green)),
+      //
+      // and then invoke "hot reload" (press "r" in the console where you ran
+      // "flutter run", or simply save your changes to "hot reload" in a Flutter
+      // IDE). Notice that the counter didn't reset back to zero; the application
+      // is not restarted.
+      theme: ThemeData.from(colorScheme: ColorScheme.light()),
```

Comment:
I think we need to have this be simpler, e.g.:
```
theme: ThemeData.baseline(
  ColorScheme's arguments
  textTheme
  various other themes like chipTheme et al
),
theme: ThemeData.baselineDark(
  same as above, but different defaults
),
```

That way the change becomes:
```
theme: ThemeData.baseline(primary: Colors.blue),
```
...to:
```
theme: ThemeData.baseline(primary: Colors.green),
```
...and the dark theme is:
```
darkTheme: ThemeData.baselineDark(),
```